# This srv file defines the service
# to detect all potential invalid arrangement states
# given the start and the final arrangement

int32[] start_arrangement
int32[] target_arrangement
---
bool success